contracts/interfaces/i_emission_manager.cairo

```diff
@@ -10,14 +10,15 @@ namespace IEmissionManager:
     end
     func set_emission_per_second():
     end
-    func set_claimer():
+    func set_claimer(_user : felt, _claimer : felt):
     end
+    # Ask why it was done this way. (underscore)
```

Comment:
you mean underscore as prefix of arguments `_user` and `_claimer`? we don't use underscore as prefix in the code base yet

Reply:
There is a variable that had an underscore as a suffix (rewards_controller_), I was wondering the reason.